docs/conf.py

```diff
@@ -1,274 +1,66 @@
-#
-# Willow documentation build configuration file, created by
-# sphinx-quickstart on Tue Jan 14 17:38:55 2014.
-#
-# This file is execfile()d with the current directory set to its
-# containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
-
 import os
 import sys
 
-from willow import __version__
-
-# on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
-on_rtd = os.environ.get("READTHEDOCS", None) == "True"
+import sphinx_wagtail_theme
 
-if not on_rtd:  # only import and set the theme if we're building docs locally
-    import sphinx_rtd_theme
-
-    html_theme = "sphinx_rtd_theme"
-    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+from willow import __version__
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath(".."))
 
-# -- General configuration ------------------------------------------------
+# -- Project information -----------------------------------------------------
 
-# If your documentation needs a minimal Sphinx version, state it here.
-# needs_sphinx = '1.0'
+project = "Willow"
+copyright = "2014-present, Torchbox"
+author = "Torchbox"
+release = __version__
+version = __version__
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
 extensions = [
     "sphinx.ext.autodoc",
+    "sphinxcontrib.spelling",
+    "sphinx.ext.intersphinx",
+    "sphinx_copybutton",
 ]
 
-if not on_rtd:
-    extensions.append("sphinxcontrib.spelling")
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3/", None),
+}
 
-# Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
+exclude_patterns = ["_build", ".DS_Store"]
+
+html_theme = "sphinx_wagtail_theme"
+html_theme_path = [sphinx_wagtail_theme.get_html_theme_path()]
+html_theme_options = {
+    "project_name": "Willow",
+    # "logo": "",
+    "github_url": "https://github.com/wagtail/Willow/tree/main/docs",
```

Comment:
**Suggestion:** add a trailing slash to generate the correct urls. Without it, you'll get urls like `https://github.com/wagtail/Willow/blob/main/docsinstallation.rst`
```suggestion
    "github_url": "https://github.com/wagtail/Willow/tree/main/docs/",
```

Reply:
Going to https://github.com/wagtail/Willow/tree/main/docs doesn't redirect to https://github.com/wagtail/Willow/tree/main/docs/ so both are valid?

Reply:
When I build the docs locally and click the 'edit on github' button, it takes me to a 404 page. This happens because incorrect urls are generated. Adding a trailing slash to the url like suggested resolves the issue.

Reply:
In case it is not clear: Sphinx uses the `github_url` setting to generate links for the 'edit on github' button.

Reply:
ah.. good to know. sorted